model_utils/tracker.py

```diff
@@ -340,7 +340,7 @@ def finalize_class(self, sender, **kwargs):
             wrapped_descriptor = wrapper_cls(field_name, descriptor, self.attname)
             setattr(sender, field_name, wrapped_descriptor)
         self.field_map = self.get_field_map(sender)
-        models.signals.post_init.connect(self.initialize_tracker)
+        self.patch_init(sender)
         self.model_class = sender
```

Comment:
We could safely remove this property since not using signal anymore, refer to https://github.com/jazzband/django-model-utils/commit/98f078d718aca346d26a38eb1efc79b1be3598c9